doc/changes/changes_1.8.0.md

```diff
@@ -0,0 +1,33 @@
+# 1.8.0 - 2025-08-14
```

Comment:
did minor release as:
- upgrade of actions/download-artifact does not affect standard usage of PTB
- pytest is only at 8.4.4 in the poetry.lock, we still support 7.x.x in the pyproject.toml
- prysk was an internal python-toolbox dependency
   - of my locally checked out 36 repos, i used `find . -name "*.t"` to find additional prysk uses. only `error-reporting-python` used it, but it is correctly listed as its own dependency in the [pyproject.toml](https://github.com/exasol/error-reporting-python/blob/main/pyproject.toml#L27)